core-management/core-management-subsystem/src/main/java/org/wildfly/extension/core/management/CoreManagementRootResourceDefinition.java

```diff
@@ -38,7 +54,40 @@ public Collection<AttributeDefinition> getAttributes() {
     @Override
     protected List<? extends PersistentResourceDefinition> getChildren() {
         return Arrays.asList(ConfigurationChangeResourceDefinition.INSTANCE,
-                new ProcessStateListenerResourceDefinition()
+                new ProcessStateListenerResourceDefinition(),
+                UnstableApiAnnotationResourceDefinition.INSTANCE
         );
     }
+
+    private static class CoreManagementAddHandler extends AbstractBoottimeAddStepHandler {
+
+        @Override
+        protected void performBoottime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+
+            if (context.isNormalServer()) {
+                context.addStep(new AbstractDeploymentChainStep() {
+                    @Override
+                    protected void execute(DeploymentProcessorTarget processorTarget) {
+                        processorTarget.addDeploymentProcessor(SUBSYSTEM_NAME, PARSE, PARSE_SCAN_EXPERIMENTAL_ANNOTATIONS,
+                                new ScanUnstableApiAnnotationsProcessor(context.getRunningMode(), context.getStability(), UnstableApiAnnotationService.LEVEL_SUPPLIER));
+                        processorTarget.addDeploymentProcessor(SUBSYSTEM_NAME, PARSE, PARSE_REPORT_EXPERIMENTAL_ANNOTATIONS,
+                                new ReportUnstableApiAnnotationsProcessor(UnstableApiAnnotationService.LEVEL_SUPPLIER));
```

Comment:
You can probably avoid the Supplier indirection by moving DUP registration until after the UnstableApiAnnotationLevel is read.